tests/cephadm/test_prepare_host.py

```diff
@@ -0,0 +1,25 @@
+from utility.log import Log
+
+log = Log(__name__)
+
+
+class PrepareHost(Exception):
+    pass
+
+
+def run(ceph_cluster, **kw):
+    """Verify prepare host command
+    Args:
+        **kw: Key/value pairs of configuration information to be used in the test.
+    """
+    config = kw.get("config")
+    node = ceph_cluster.get_nodes(role="mon")[0]
+
+    # run prepare host command
+    log.info("Running cephadm prepare-host command")
+    cmd = "cephadm prepare-host"
```

Comment:
Instead add this command in `cli/cephadm/cephadm.py` library

Reply:
Thanks @vamahaja
Updated